src/components/BotBuilder/BotBuilder.js

```diff
@@ -111,7 +111,23 @@ class BotBuilder extends React.Component {
     if (bots) {
       bots.forEach(bot => {
         let imageUrl;
-        if (bot.image !== 'images/<image_name>') {
+        if (bot.image === 'images/<image_name>') {
+          imageUrl = `${window.location.protocol}//${
+            window.location.host
+          }/customAvatars/1.png`;
+        } else if (bot.image === 'images/<image_name_event>') {
+          imageUrl = `${window.location.protocol}//${
+            window.location.host
+          }/botTemplates/event-registration.jpg`;
+        } else if (bot.image === 'images/<image_name_job>') {
+          imageUrl = `${window.location.protocol}//${
+            window.location.host
+          }/botTemplates/job-application.jpg`;
+        } else if (bot.image === 'images/<image_name_contact>') {
+          imageUrl = `${window.location.protocol}//${
+            window.location.host
+          }/botTemplates/contact-us.png`;
+        } else {
           imageUrl = bot.image
             ? BASE_URL +
               '/cms/getImage.png?access_token=' +
```

Comment:
Can't we use templates here too? @AdityaSrivast

Reply:
Yes @AakashMallik missed it out